scripts/db-connect.sh

```diff
@@ -75,12 +75,11 @@ case $1 in
     * ) error "ENV must be one of dev, alpha, or staging";;
 esac
 
-VAULT_PATH="secret/dsde/firecloud/$1/consent/secrets/postgres/app_sql_user"
-VAULT_JSON=$(vault read -format=json "$VAULT_PATH")
+PG_JSON=$(gcloud secrets versions access latest --secret=consent-postgres-creds)
```

Comment:
I think you need to map env (`$1`) to a google project and pass `--project <>`.

Reply:
The problem isn't the path, the problem is that the current default google project may not be correct for the env that the user is requesting. In order to give the user the correct password and instance name, the command needs a `--project` argument. For example, see how this is done in the TDR script https://github.com/DataBiosphere/jade-data-repo/blob/develop/ops/db-connect.sh#L129